aten/src/ATen/native/native_functions.yaml

```diff
@@ -3691,15 +3691,26 @@
 - func: prelu(Tensor self, Tensor weight) -> Tensor
   variants: function, method
   dispatch:
+    MkldnnCPU: mkldnn_prelu
     CPU: prelu_cpu
     CUDA: prelu_cuda
 
 - func: prelu_backward(Tensor grad_output, Tensor self, Tensor weight) -> (Tensor, Tensor)
   variants: function, method
   dispatch:
+    MkldnnCPU: mkldnn_prelu_backward
     CPU: prelu_backward_cpu
     CUDA: prelu_backward_cuda
 
+- func: mkldnn_prelu(Tensor self, Tensor weight) -> Tensor
```

Comment:
why do we need these functions? MkldnnPrelu can dispatch regular `prelu` to mkldnn

Reply:
Hi @ngimel , these functions are for module `MkldnnPrelu` to find the correct functions when doing `to_mkldnn()`. https://github.com/pytorch/pytorch/blob/e3fc20f676d97348c02647295b2dbf3aca878a2f/torch/utils/mkldnn.py#L200
https://github.com/pytorch/pytorch/blob/e3fc20f676d97348c02647295b2dbf3aca878a2f/test/test_mkldnn.py#L409

Reply:
Those functions should be correctly selected during dispatch on mkldnn tensors, they shouldn't be separate functions in native_functions.yaml

Reply:
Got it. I have removed `mkldnn_prelu` in native_functions.yaml. And use `torch.prelu` to call the function, and it can be correctly selected during dispatch.